src/theory/arrays/theory_arrays.cpp

```diff
@@ -1184,19 +1170,7 @@ void TheoryArrays::presolve()
 
 Node TheoryArrays::getSkolem(TNode ref)
 {
-  // the call to SkolemCache::getExtIndexSkolem should be deterministic, but use
-  // cache anyways for now
-  Node skolem;
-  std::unordered_map<Node, Node>::iterator it = d_skolemCache.find(ref);
```

Comment:
Do we still need d_skolemCache after this change?

Reply:
No, it is deleted with this PR.

Reply:
Yes, I missed that.